test/unittest/dds/subscriber/DataReaderTests.cpp

```diff
@@ -57,6 +57,9 @@
 
 #include "../../logging/mock/MockConsumer.h"
 
+#include <rtps/transport/test_UDPv4Transport.h>
```

Comment:
This is adding an unnecessary dependency on asio (and thus the build fails), but only the transport descriptor is necessary:

```suggestion
#include <fastdds/rtps/transport/test_UDPv4TransportDescriptor.h>
```

Reply:
I need access to `rtps::test_UDPv4Transport::test_UDPv4Transport_ShutdownAllNetwor` in line 1974.

Reply:
You could achieve the same with `percentageOfMessagesToDrop` or `messages_filter_` on `test_UDPv4TransportDescriptor`

Reply:
Done